.tmuxinatorConfigs/testing.yml

```diff
@@ -0,0 +1,13 @@
+# /home/$(USER)/.config/tmuxinator/testing.yml
+# Testing
```

Comment:
Can you maybe add a comment explaining what this configuration does? I can't understand without needing to launch the actual configuration myself.

I'm confused on:
1. Why do we have a "testing" configuration? Does this mean "testing" as in "testing at Lake Wauburg", "testing that `tmuxinator` works", or an accidental test file that was left in this PR?
2. What is this config file supposed to accomplish? Is this a config file that is ready to go to be used all the time, or should we just launch this file once to show that the program works?

Reply:
1. This is a base file, probably better called example.
2. As per above, basically just showing how to create a simple setup. This would need to be extended by Alex or you with whatever tools you want available.